ghost/posts-service/lib/PostsService.js

```diff
@@ -365,6 +366,58 @@ class PostsService {
 
         return cacheInvalidate;
     }
+
+    async copyPost(frame) {
+        const existingPost = await this.models.Post.findOne({
+            id: frame.options.id,
+            status: 'all'
+        }, frame.options);
+
+        const newPostData = omit(
+            existingPost.attributes,
+            [
```

Comment:
I'm slightly cautious when using omit in code as it hides additional maintenance long-term. For example, when we add a new field to the `posts` table we'd have remember to exclude it here too if it's a field we don't want to copy over. On the other hand, we'd get a failure for the snapshot test and would need to update it, so there is some signal for us to check on this list in the future. What are your thoughts on this @mike182uk ?